src/components/FileInput.js

```diff
@@ -78,7 +79,23 @@ class FileInput extends Component {
     const reader = new FileReader();
 
     reader.readAsDataURL(file);
-    reader.onload = event => this.setState({ image: event.target.result });
+    reader.onload = event => {
+      const ratio = this.props.scaleImageOptions
+        && apsectRatio.crop(
```

Comment:
If we're writing a library I don't think we should use a library that we cannot maintain (and that it hasn't been updated for the last three years) to perform a fairly simple calculation.